src/extensions/CoveoSearch/CoveoSearchDialog.js

```diff
@@ -4,7 +4,13 @@
 import React, { useState, useEffect, useCallback, useRef } from "react";
 import ReactDom from "react-dom";
 // import PropTypes from 'prop-types';
```

Comment:
remove if not needed

Reply:
Left it in since we want to add these back. Just wanted to get something up and running without spending too much time on boilerplate stuff.